@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts

```diff
@@ -25,6 +25,219 @@
 
 import { RequestBuilder, RequestOptions, UrlBuilder } from "./RequestBuilder";
 
+export interface PatchProject {
+    /**
+     * The name of the Project
```

Comment:
```suggestion
     * The name of the Project.
```

Same for all other doxygen descriptions below. All should be `.` terminated.

Reply:
added to all lines, thanks